### DISCUSSION TITLE
Smarty: Invalid compiled template for 'validating_login.tpl' error

### DISCUSSION THREAD

**Discussion by mtbluedog** [geodesicsolutions-community/geocore-community#224] [Q&A] [answered]
Using GeoCore Ver. 18.02.0 (license patch installed), I still have a login issue when going to example.com/admin/ . I get the login screen, but when I log in, I get the error

`--> Smarty: Invalid compiled template for 'validating_login.tpl' <-- `

The file /admin/templates/validating_login.tpl existed, but was empty, so I replaced it from a v20 download, knowing the version difference might be a problem, and no luck with fixing the error.

No errors in the PHP error logs.

I'm thinking other files are damaged or missing. Is it possible to download a full copy of 18.02.0 so I can replace the entire admin directory?

With a text editor, I compared all the files in the admin directory in both my current v18 and a v20 download, and the file differences are numerous.

Is this a known issue with GoDaddy economy hosting?

**Comment by vicos59** [comment, marked as answer]
Didn't the other guy provide another template for Godaddy in the other thread you had open? 

No, you can't assume a v20 file is the same as the v18 file. All of the code is available here in the code section where you can see the revision  history.

**Reply by mtbluedog** [reply]
He mentioned there were known problems with GoDaddy and Ioncube, and the patch removes the need for Ioncube.
He gave me an updated link for the license removal patch for v18.

**Reply by vicos59** [reply]
I'm pretty sure he uploaded a new template (.tpl) for you that was just for Godaddy hosted sites that fixed a problem. I don't have time to look thru all the discussions, so I would go back and look at that one again.

**Comment by blufyremedia** [comment]
Here is a copy of the original v18.02 validating_login tpl file. This is not related to Ioncube. We've only had reports of this file being corrupted on GoDaddy sites, but to understand what transpired in your case, you would need to discuss that further with your host (GoDaddy).

[validating_login.zip](https://github.com/geodesicsolutions-community/geocore-community/files/13922535/validating_login.zip)

**Reply by mtbluedog** [reply]
Thanks! That turns out to be the same as the v20 file. But I still get the error.
